LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/Region.cs

```diff
@@ -354,6 +354,11 @@ public DataRateIndex GetDRFromFreqAndChan(string datr)
             return DRtoConfiguration.FirstOrDefault(x => x.Value.DataRate.XpkDatr == datr).Key;
         }
 
+        public DataRateIndex GetDRFromFreqAndChan(DataRate datr)
```

Comment:
Is this the best name for this method? Why not simply call it `GetDataRateIndex`?